### ISSUE TITLE
Extremely frequent hangs since upgrading to Big Sur; IINA basically unusable

### ISSUE THREAD

**Issue by abatty** [iina/iina#3364]
**System and IINA version:**

- macOS 11.2.3
- IINA 1.1.2

**Expected behavior:**
Play a video (.mp4, .avi) with correct dimensions and without hanging (beachball), requiring force quit (not responding).

**Actual behavior:**
When I open a problem file, usually the video dimensions are wrong, often which it stretched vertically and a black bar on one side of the playback window. Sometimes the resolution is just totally wrong, like a square or near to it. 

The video and audio both play, but the application becomes unresponsive, with a spinning beach ball. IINA is listed in the Force Quit window as "not responding," and must be Force Quit. Only once have I gotten a crash log from the OS when I've done this; I believe it is the one pasted below, and it classifies it as a "hang."

<details>
<summary><b>Crash report:</b></summary>

```
Date/Time:        2021-03-20 11:08:29.014 +0900
End time:         2021-03-20 11:08:55.922 +0900
OS Version:       macOS 11.2.3 (Build 20D91)
Architecture:     x86_64h
Report Version:   32
Share With Devs:  Yes

Data Source:      Stackshots
Shared Cache:     52762DF1-0E55-3F8B-9588-8169E6D0EB95 slid base address 0x7fff200c2000, slide 0xc2000

Command:          IINA
Path:             /Applications/IINA.app/Contents/MacOS/IINA
Identifier:       com.colliderli.iina
Version:          1.1.2 (128)
Parent:           launchd [1]
PID:              27564
Time Since Fork:  27s

Event:            hang
Duration:         26.91s
Duration Sampled: 1.50s (process was unresponsive for 25 seconds before sampling)
Steps:            15 (100ms sampling interval)
Report threshold: 2.5s

Hardware model:   Macmini8,1
Active cpus:      12
Boot args:        chunklist-security-epoch=0 -chunklist-no-rev2-dev

Time Awake Since Boot: 89503s
Time Since Wake:  7575s

Fan speed:        2254 rpm
```
</details>

<br>

<details>
<summary><b>mpv log:</b></summary>

```
[   0.009][v][cplayer] mpv 0.32.0 Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
[   0.010][v][cplayer]  built on Sun Nov  8 00:37:38 CST 2020
[   0.010][v][cplayer] ffmpeg library versions:
[   0.010][v][cplayer]    libavutil       56.51.100
[   0.010][v][cplayer]    libavcodec      58.91.100
[   0.010][v][cplayer]    libavformat     58.45.100
[   0.010][v][cplayer]    libswscale      5.7.100
[   0.010][v][cplayer]    libavfilter     7.85.100
[   0.010][v][cplayer]    libswresample   3.7.100
[   0.010][v][cplayer] ffmpeg version: 4.3.1
[   0.010][v][cplayer] 
[   0.010][v][cplayer] Configuration: waf configure --prefix=/usr/local/Cellar/mpv-iina/0.32.0 --enable-javascript --enable-libmpv-shared --enable-lua --enable-libarchive --enable-uchardet --enable-libbluray --disable-swift --disable-debug-build --disable-macos-media-player --confdir=/usr/local/etc/mpv --datadir=/usr/local/Cellar/mpv-iina/0.32.0/share/mpv-iina --mandir=/usr/local/Cellar/mpv-iina/0.32.0/share/man --docdir=/usr/local/Cellar/mpv-iina/0.32.0/share/doc/mpv-iina --lua=51deb
[   0.010][v][cplayer] List of enabled features: 51deb aligned_alloc asm atomics bsd-fstatfs build-date cocoa coreaudio cplayer cplugins drmprime fchmod ffmpeg gl gl-cocoa glob glob-posix gnuc gpl iconv javascript jpeg lcms2 libarchive libass libass-osd libav-any libavcodec libavdevice libavutil libbluray libdl libm libmpv-shared lua macos-10-11-features macos-10-12-2-features macos-10-14-features macos-touchbar optimize osx-thread-name plain-gl posix posix-or-mingw posix-spawn posix-spawn-native pthreads rubberband stdatomic uchardet videotoolbox-gl videotoolbox-hwaccel zlib
[   0.010][v][cplayer] mpv 0.32.0 Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
[   0.010][v][cplayer]  built on Sun Nov  8 00:37:38 CST 2020
[   0.010][v][cplayer] ffmpeg library versions:
[   0.010][v][cplayer]    libavutil       56.51.100
[   0.010][v][cplayer]    libavcodec      58.91.100
[   0.010][v][cplayer]    libavformat     58.45.100
[   0.010][v][cplayer]    libswscale      5.7.100
[   0.010][v][cplayer]    libavfilter     7.85.100
[   0.010][v][cplayer]    libswresample   3.7.100
[   0.010][v][cplayer] ffmpeg version: 4.3.1
[   0.010][v][cplayer] 
[   0.010][v][cplayer] Configuration: waf configure --prefix=/usr/local/Cellar/mpv-iina/0.32.0 --enable-javascript --enable-libmpv-shared --enable-lua --enable-libarchive --enable-uchardet --enable-libbluray --disable-swift --disable-debug-build --disable-macos-media-player --confdir=/usr/local/etc/mpv --datadir=/usr/local/Cellar/mpv-iina/0.32.0/share/mpv-iina --mandir=/usr/local/Cellar/mpv-iina/0.32.0/share/man --docdir=/usr/local/Cellar/mpv-iina/0.32.0/share/doc/mpv-iina --lua=51deb
[   0.010][v][cplayer] List of enabled features: 51deb aligned_alloc asm atomics bsd-fstatfs build-date cocoa coreaudio cplayer cplugins drmprime fchmod ffmpeg gl gl-cocoa glob glob-posix gnuc gpl iconv javascript jpeg lcms2 libarchive libass libass-osd libav-any libavcodec libavdevice libavutil libbluray libdl libm libmpv-shared lua macos-10-11-features macos-10-12-2-features macos-10-14-features macos-touchbar optimize osx-thread-name plain-gl posix posix-or-mingw posix-spawn posix-spawn-native pthreads rubberband stdatomic uchardet videotoolbox-gl videotoolbox-hwaccel zlib
[   0.010][v][bdmv/bluray] Opening /Applications/IINA.app/Contents/Resources/config/iina-default-input.conf
[   0.010][v][file] Opening /Applications/IINA.app/Contents/Resources/config/iina-default-input.conf
[   0.010][d][file] resize stream to 131072 bytes
[   0.010][d][file] Stream opened successfully.
[   0.010][v][input] Parsing input config file /Applications/IINA.app/Contents/Resources/config/iina-default-input.conf
[   0.011][v][input] Input config file /Applications/IINA.app/Contents/Resources/config/iina-default-input.conf parsed: 69 binds
[   0.011][d][global] user path: '/Users/user/Library/Logs/com.colliderli.iina/2021-03-20-11-51-13_NyIkwr/mpv.log' -> '/Users/user/Library/Logs/com.colliderli.iina/2021-03-20-11-51-13_NyIkwr/mpv.log'
[   0.011][d][ytdl_hook] Loading lua script @ytdl_hook.lua...
[   0.011][d][stats] Loading lua script @stats.lua...
[   0.011][d][console] Loading lua script @console.lua...
[   0.011][d][osx] Exiting...
[   0.011][v][cplayer] Set property: vo="libmpv" -> 1
[   0.011][d][ytdl_hook] loading mp.defaults
[   0.011][d][stats] loading mp.defaults
[   0.011][d][console] loading mp.defaults
[   0.011][v][cplayer] Set property: keepaspect="no" -> 1
[   0.011][v][cplayer] Set property: gpu-hwdec-interop="auto" -> 1
[   0.011][v][cplayer] Waiting for scripts...
[   0.012][d][stats] loading @stats.lua
[   0.012][d][ytdl_hook] loading @ytdl_hook.lua
[   0.012][d][console] loading @console.lua
[   0.012][d][ytdl_hook] reading options for ytdl_hook 
[   0.012][d][ytdl_hook] script-opts/ytdl_hook.conf not found. 
[   0.013][d][ytdl_hook] lua-settings/ytdl_hook.conf not found. 
[   0.013][d][stats] reading options for stats 
[   0.013][d][stats] script-opts/stats.conf not found. 
[   0.013][d][stats] lua-settings/stats.conf not found. 
[   0.035][v][ao/coreaudio] selected audio output device: USB Audio CODEC  (86)
[   0.038][d][console] reading options for console 
[   0.038][d][console] script-opts/console.conf not found. 
[   0.038][d][console] lua-settings/console.conf not found. 
[   0.039][d][cplayer] Run command: define-section, flags=64, args=["input_stats", "i script-binding stats/display-stats\nI script-binding stats/display-stats-toggle\n", "default"]
[   0.039][d][cplayer] Run command: enable-section, flags=64, args=["input_stats", "allow-hide-cursor+allow-vo-dragging"]
[   0.039][d][cplayer] Run command: define-section, flags=64, args=["input_forced_stats", "", "force"]
[   0.039][d][cplayer] Run command: enable-section, flags=64, args=["input_forced_stats", "allow-hide-cursor+allow-vo-dragging"]
[   0.042][v][cplayer] Done loading scripts.
[   0.043][v][osd/libass] Shaper: FriBidi 1.0.10 (SIMPLE) HarfBuzz-ng 2.7.2 (COMPLEX)
[   0.043][v][osd/libass] Setting up fonts...
[   0.044][v][osd/libass] Using font provider coretext
[   0.044][v][osd/libass] Done.
[   0.044][d][cplayer] Run command: define-section, flags=64, args=["input_console", "", "default"]
[   0.044][d][cplayer] Run command: enable-section, flags=64, args=["input_console", "allow-hide-cursor+allow-vo-dragging"]
[   0.044][d][cplayer] Run command: define-section, flags=64, args=["input_forced_console", "", "force"]
[   0.044][d][cplayer] Run command: enable-section, flags=64, args=["input_forced_console", "allow-hide-cursor+allow-vo-dragging"]
[   0.203][v][libmpv_render] GL_VERSION='4.1 INTEL-16.1.12'
[   0.203][v][libmpv_render] Detected desktop OpenGL 4.1.
[   0.203][v][libmpv_render] GL_VENDOR='Intel Inc.'
[   0.203][v][libmpv_render] GL_RENDERER='Intel(R) UHD Graphics 630'
[   0.203][v][libmpv_render] GL_SHADING_LANGUAGE_VERSION='4.10'
[   0.203][d][libmpv_render] Combined OpenGL extensions string:
[   0.203][d][libmpv_render]  GL_ARB_blend_func_extended GL_ARB_draw_buffers_blend GL_ARB_draw_indirect GL_ARB_ES2_compatibility GL_ARB_explicit_attrib_location GL_ARB_gpu_shader_fp64 GL_ARB_gpu_shader5 GL_ARB_instanced_arrays GL_ARB_internalformat_query GL_ARB_occlusion_query2 GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_separate_shader_objects GL_ARB_shader_bit_encoding GL_ARB_shader_subroutine GL_ARB_shading_language_include GL_ARB_tessellation_shader GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_cube_map_array GL_ARB_texture_gather GL_ARB_texture_query_lod GL_ARB_texture_rgb10_a2ui GL_ARB_texture_storage GL_ARB_texture_swizzle GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_vertex_attrib_64bit GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_viewport_array GL_EXT_debug_label GL_EXT_debug_marker GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_texture_compression_s3tc GL_EXT_texture_filter_anisotropic GL_EXT_texture_sRGB_decode GL_APPLE_client_storage GL_APPLE_container_object_shareable GL_APPLE_flush_render GL_APPLE_object_purgeable GL_APPLE_rgb_422 GL_APPLE_row_bytes GL_APPLE_texture_range GL_ATI_texture_mirror_once GL_NV_texture_barrier
[   0.203][v][libmpv_render] Loaded extension GL_APPLE_rgb_422.
[   0.203][v][libmpv_render] GL_*_swap_control extension missing.
[   0.203][d][libmpv_render] Texture formats:
[   0.203][d][libmpv_render]   NAME       COMP*TYPE SIZE           DEPTH PER COMP.
[   0.203][d][libmpv_render]   r8         1*unorm   1B    LF CR ST {8}
[   0.203][d][libmpv_render]   rg8        2*unorm   2B    LF CR ST {8 8}
[   0.203][d][libmpv_render]   rgb8       3*unorm   3B    LF CR ST {8 8 8}
[   0.203][d][libmpv_render]   rgba8      4*unorm   4B    LF CR ST {8 8 8 8}
[   0.203][d][libmpv_render]   r16        1*unorm   2B    LF CR ST {16}
[   0.203][d][libmpv_render]   rg16       2*unorm   4B    LF CR ST {16 16}
[   0.203][d][libmpv_render]   rgb16      3*unorm   6B    LF CR ST {16 16 16}
[   0.203][d][libmpv_render]   rgba16     4*unorm   8B    LF CR ST {16 16 16 16}
[   0.203][d][libmpv_render]   r8ui       1*uint    1B       CR ST {8}
[   0.203][d][libmpv_render]   rg8ui      2*uint    2B       CR ST {8 8}
[   0.203][d][libmpv_render]   rgb8ui     3*uint    3B          ST {8 8 8}
[   0.203][d][libmpv_render]   rgba8ui    4*uint    4B       CR ST {8 8 8 8}
[   0.203][d][libmpv_render]   r16ui      1*uint    2B       CR ST {16}
[   0.203][d][libmpv_render]   rg16ui     2*uint    4B       CR ST {16 16}
[   0.203][d][libmpv_render]   rgb16ui    3*uint    6B          ST {16 16 16}
[   0.203][d][libmpv_render]   rgba16ui   4*uint    8B       CR ST {16 16 16 16}
[   0.203][d][libmpv_render]   r16f       1*float   4B    LF CR ST {32/16}
[   0.203][d][libmpv_render]   rg16f      2*float   8B    LF CR ST {32/16 32/16}
[   0.203][d][libmpv_render]   rgb16f     3*float  12B    LF CR ST {32/16 32/16 32/16}
[   0.203][d][libmpv_render]   rgba16f    4*float  16B    LF CR ST {32/16 32/16 32/16 32/16}
[   0.203][d][libmpv_render]   r32f       1*float   4B    LF CR ST {32}
[   0.203][d][libmpv_render]   rg32f      2*float   8B    LF CR ST {32 32}
[   0.203][d][libmpv_render]   rgb32f     3*float  12B    LF CR ST {32 32 32}
[   0.203][d][libmpv_render]   rgba32f    4*float  16B    LF CR ST {32 32 32 32}
[   0.203][d][libmpv_render]   rgb10_a2   4*unorm   4B    LF CR ST {0/10 0/10 0/10 0/2}
[   0.203][d][libmpv_render]   rgba12     4*unorm   8B    LF CR ST {16 16 16 16}
[   0.203][d][libmpv_render]   rgb10      3*unorm   6B    LF CR ST {16 16 16}
[   0.203][d][libmpv_render]   rgb565     3*unorm   2B    LF    ST {0/8 0/8 0/8}
[   0.203][d][libmpv_render]   appleyp    0*unorm   2B    LF    ST {}
[   0.203][d][libmpv_render]  LA = LUMINANCE_ALPHA hack format
[   0.203][d][libmpv_render]  LF = linear filterable
[   0.203][d][libmpv_render]  CR = can be used for render targets
[   0.203][d][libmpv_render]  ST = can be used for storable images
[   0.203][d][libmpv_render] Image formats:
[   0.203][d][libmpv_render]   yuv444p => 3 planes 1x1 8/0 [r8/r8/r8] (r/g/b)
[   0.203][d][libmpv_render]   yuv420p => 3 planes 2x2 8/0 [r8/r8/r8] (r/g/b)
[   0.203][d][libmpv_render]   gray => 1 planes 1x1 8/0 [r8] (r)
[   0.203][d][libmpv_render]   gray16 => 1 planes 1x1 16/0 [r16] (r)
[   0.203][d][libmpv_render]   uyvy422 => 1 planes 1x1 0/0 [appleyp] (brg)
[   0.203][d][libmpv_render]   nv12 => 2 planes 2x2 8/0 [r8/rg8] (r/gb)
[   0.203][d][libmpv_render]   p010 => 2 planes 2x2 16/6 [r16/rg16] (r/gb)
[   0.203][d][libmpv_render]   nv24 => 2 planes 1x1 8/0 [r8/rg8] (r/gb)
[   0.203][d][libmpv_render]   argb => 1 planes 1x1 8/0 [rgba8] (argb)
[   0.203][d][libmpv_render]   bgra => 1 planes 1x1 8/0 [rgba8] (bgra)
[   0.203][d][libmpv_render]   abgr => 1 planes 1x1 8/0 [rgba8] (abgr)
[   0.203][d][libmpv_render]   rgba => 1 planes 1x1 8/0 [rgba8] (rgba)
[   0.203][d][libmpv_render]   bgr24 => 1 planes 1x1 8/0 [rgb8] (bgr)
[   0.203][d][libmpv_render]   rgb24 => 1 planes 1x1 8/0 [rgb8] (rgb)
[   0.203][d][libmpv_render]   0rgb => 1 planes 1x1 8/0 [rgba8] (_rgb)
[   0.203][d][libmpv_render]   bgr0 => 1 planes 1x1 8/0 [rgba8] (bgr)
[   0.203][d][libmpv_render]   0bgr => 1 planes 1x1 8/0 [rgba8] (_bgr)
[   0.203][d][libmpv_render]   rgb0 => 1 planes 1x1 8/0 [rgba8] (rgb)
[   0.203][d][libmpv_render]   rgba64 => 1 planes 1x1 16/0 [rgba16] (rgba)
[   0.203][d][libmpv_render]   rgb565 => 1 planes 1x1 0/0 [rgb565] (rgb)
[   0.203][d][libmpv_render]   rgb30 => 1 planes 1x1 10/0 [rgb10_a2] (rgb)
[   0.203][d][libmpv_render]   vdpau
[   0.203][d][libmpv_render]   vdpau_output
[   0.203][d][libmpv_render]   vaapi
[   0.203][d][libmpv_render]   dxva2_vld
[   0.203][d][libmpv_render]   mmal
[   0.203][d][libmpv_render]   videotoolbox
[   0.203][d][libmpv_render]   drm_prime
[   0.203][d][libmpv_render]   yuyv422
[   0.203][d][libmpv_render]   yuv422p => 3 planes 2x1 8/0 [r8/r8/r8] (r/g/b)
[   0.203][d][libmpv_render]   yuv410p => 3 planes 4x4 8/0 [r8/r8/r8] (r/g/b)
[   0.203][d][libmpv_render]   yuv411p => 3 planes 4x1 8/0 [r8/r8/r8] (r/g/b)
[   0.203][d][libmpv_render]   monow
[   0.203][d][libmpv_render]   monob
[   0.203][d][libmpv_render]   pal8
[   0.203][d][libmpv_render]   yuvj422p => 3 planes 2x1 8/0 [r8/r8/r8] (r/g/b)
[   0.203][d][libmpv_render]   uyyvyy411
[   0.203][d][libmpv_render]   bgr8
[   0.203][d][libmpv_render]   bgr4
[   0.203][d][libmpv_render]   bgr4_byte
[   0.203][d][libmpv_render]   rgb8
[   0.203][d][libmpv_render]   rgb4
[   0.203][d][libmpv_render]   rgb4_byte
[   0.203][d][libmpv_render]   nv21 => 2 planes 2x2 8/0 [r8/rg8] (r/bg)
[   0.203][d][libmpv_render]   gray16be
[   0.203][d][libmpv_render]   yuv440p => 3 planes 1x2 8/0 [r8/r8/r8] (r/g/b)
[   0.203][d][libmpv_render]   yuvj440p => 3 planes 1x2 8/0 [r8/r8/r8] (r/g/b)
[   0.203][d][libmpv_render]   yuva420p => 4 planes 2x2 8/0 [r8/r8/r8/r8] (r/g/b/a)
[   0.203][d][libmpv_render]   rgb48be
[   0.203][d][libmpv_render]   rgb48 => 1 planes 1x1 16/0 [rgb16] (rgb)
[   0.203][d][libmpv_render]   rgb565be
[   0.203][d][libmpv_render]   rgb555be
[   0.203][d][libmpv_render]   rgb555
[   0.203][d][libmpv_render]   bgr565be
[   0.203][d][libmpv_render]   bgr565
[   0.203][d][libmpv_render]   bgr555be
[   0.203][d][libmpv_render]   bgr555
[   0.203][d][libmpv_render]   vaapi_moco
[   0.203][d][libmpv_render]   vaapi_idct
[   0.203][d][libmpv_render]   yuv420p16 => 3 planes 2x2 16/0 [r16/r16/r16] (r/g/b)
[   0.203][d][libmpv_render]   yuv420p16be
[   0.203][d][libmpv_render]   yuv422p16 => 3 planes 2x1 16/0 [r16/r16/r16] (r/g/b)
[   0.203][d][libmpv_render]   yuv422p16be
[   0.203][d][libmpv_render]   yuv444p16 => 3 planes 1x1 16/0 [r16/r16/r16] (r/g/b)
[   0.203][d][libmpv_render]   yuv444p16be
[   0.203][d][libmpv_render]   rgb444
[   0.203][d][libmpv_render]   rgb444be
[   0.203][d][libmpv_render]   bgr444
[   0.203][d][libmpv_render]   bgr444be
[   0.203][d][libmpv_render]   ya8 => 1 planes 1x1 8/0 [rg8] (ra)
[   0.203][d][libmpv_render]   bgr48be
[   0.203][d][libmpv_render]   bgr48 => 1 planes 1x1 16/0 [rgb16] (bgr)
[   0.203][d][libmpv_render]   yuv420p9be
[   0.204][d][libmpv_render]   yuv420p9 => 3 planes 2x2 16/-7 [r16/r16/r16] (r/g/b)
[   0.204][d][libmpv_render]   yuv420p10be
[   0.204][d][libmpv_render]   yuv420p10 => 3 planes 2x2 16/-6 [r16/r16/r16] (r/g/b)
[   0.204][d][libmpv_render]   yuv422p10be
[   0.204][d][libmpv_render]   yuv422p10 => 3 planes 2x1 16/-6 [r16/r16/r16] (r/g/b)
[   0.204][d][libmpv_render]   yuv444p9be
[   0.204][d][libmpv_render]   yuv444p9 => 3 planes 1x1 16/-7 [r16/r16/r16] (r/g/b)
[   0.204][d][libmpv_render]   yuv444p10be
[   0.204][d][libmpv_render]   yuv444p10 => 3 planes 1x1 16/-6 [r16/r16/r16] (r/g/b)
[   0.204][d][libmpv_render]   yuv422p9be
[   0.204][d][libmpv_render]   yuv422p9 => 3 planes 2x1 16/-7 [r16/r16/r16] (r/g/b)
[   0.204][d][libmpv_render]   gbrp => 3 planes 1x1 8/0 [r8/r8/r8] (g/b/r)
[   0.204][d][libmpv_render]   gbrp9be
[   0.204][d][libmpv_render]   gbrp9 => 3 planes 1x1 16/-7 [r16/r16/r16] (g/b/r)
[   0.204][d][libmpv_render]   gbrp10be
[   0.204][d][libmpv_render]   gbrp10 => 3 planes 1x1 16/-6 [r16/r16/r16] (g/b/r)
[   0.204][d][libmpv_render]   gbrp16be
[   0.204][d][libmpv_render]   gbrp16 => 3 planes 1x1 16/0 [r16/r16/r16] (g/b/r)
[   0.204][d][libmpv_render]   yuva422p => 4 planes 2x1 8/0 [r8/r8/r8/r8] (r/g/b/a)
[   0.204][d][libmpv_render]   yuva444p => 4 planes 1x1 8/0 [r8/r8/r8/r8] (r/g/b/a)
[   0.204][d][libmpv_render]   yuva420p9be
[   0.204][d][libmpv_render]   yuva420p9 => 4 planes 2x2 16/-7 [r16/r16/r16/r16] (r/g/b/a)
[   0.204][d][libmpv_render]   yuva422p9be
[   0.204][d][libmpv_render]   yuva422p9 => 4 planes 2x1 16/-7 [r16/r16/r16/r16] (r/g/b/a)
[   0.204][d][libmpv_render]   yuva444p9be
[   0.204][d][libmpv_render]   yuva444p9 => 4 planes 1x1 16/-7 [r16/r16/r16/r16] (r/g/b/a)
[   0.204][d][libmpv_render]   yuva420p10be
[   0.204][d][libmpv_render]   yuva420p10 => 4 planes 2x2 16/-6 [r16/r16/r16/r16] (r/g/b/a)
[   0.204][d][libmpv_render]   yuva422p10be
[   0.204][d][libmpv_render]   yuva422p10 => 4 planes 2x1 16/-6 [r16/r16/r16/r16] (r/g/b/a)
[   0.204][d][libmpv_render]   yuva444p10be
[   0.204][d][libmpv_render]   yuva444p10 => 4 planes 1x1 16/-6 [r16/r16/r16/r16] (r/g/b/a)
[   0.204][d][libmpv_render]   yuva420p16be
[   0.204][d][libmpv_render]   yuva420p16 => 4 planes 2x2 16/0 [r16/r16/r16/r16] (r/g/b/a)
[   0.204][d][libmpv_render]   yuva422p16be
[   0.204][d][libmpv_render]   yuva422p16 => 4 planes 2x1 16/0 [r16/r16/r16/r16] (r/g/b/a)
[   0.204][d][libmpv_render]   yuva444p16be
[   0.204][d][libmpv_render]   yuva444p16 => 4 planes 1x1 16/0 [r16/r16/r16/r16] (r/g/b/a)
[   0.204][d][libmpv_render]   xyz12 => 1 planes 1x1 16/4 [rgb16] (rgb)
[   0.204][d][libmpv_render]   xyz12be
[   0.204][d][libmpv_render]   nv16 => 2 planes 2x1 8/0 [r8/rg8] (r/gb)
[   0.204][d][libmpv_render]   nv20 => 2 planes 2x1 16/-6 [r16/rg16] (r/gb)
[   0.204][d][libmpv_render]   nv20be
[   0.204][d][libmpv_render]   rgba64be
[   0.204][d][libmpv_render]   bgra64be
[   0.204][d][libmpv_render]   bgra64 => 1 planes 1x1 16/0 [rgba16] (bgra)
[   0.204][d][libmpv_render]   yvyu422
[   0.204][d][libmpv_render]   ya16be
[   0.204][d][libmpv_render]   ya16 => 1 planes 1x1 16/0 [rg16] (ra)
[   0.204][d][libmpv_render]   gbrap => 4 planes 1x1 8/0 [r8/r8/r8/r8] (g/b/r/a)
[   0.204][d][libmpv_render]   gbrap16be
[   0.204][d][libmpv_render]   gbrap16 => 4 planes 1x1 16/0 [r16/r16/r16/r16] (g/b/r/a)
[   0.204][d][libmpv_render]   qsv
[   0.204][d][libmpv_render]   d3d11va_vld
[   0.204][d][libmpv_render]   cuda
[   0.204][d][libmpv_render]   yuv420p12be
[   0.204][d][libmpv_render]   yuv420p12 => 3 planes 2x2 16/-4 [r16/r16/r16] (r/g/b)
[   0.204][d][libmpv_render]   yuv420p14be
[   0.204][d][libmpv_render]   yuv420p14 => 3 planes 2x2 16/-2 [r16/r16/r16] (r/g/b)
[   0.204][d][libmpv_render]   yuv422p12be
[   0.204][d][libmpv_render]   yuv422p12 => 3 planes 2x1 16/-4 [r16/r16/r16] (r/g/b)
[   0.204][d][libmpv_render]   yuv422p14be
[   0.204][d][libmpv_render]   yuv422p14 => 3 planes 2x1 16/-2 [r16/r16/r16] (r/g/b)
[   0.204][d][libmpv_render]   yuv444p12be
[   0.204][d][libmpv_render]   yuv444p12 => 3 planes 1x1 16/-4 [r16/r16/r16] (r/g/b)
[   0.204][d][libmpv_render]   yuv444p14be
[   0.204][d][libmpv_render]   yuv444p14 => 3 planes 1x1 16/-2 [r16/r16/r16] (r/g/b)
[   0.204][d][libmpv_render]   gbrp12be
[   0.204][d][libmpv_render]   gbrp12 => 3 planes 1x1 16/-4 [r16/r16/r16] (g/b/r)
[   0.204][d][libmpv_render]   gbrp14be
[   0.204][d][libmpv_render]   gbrp14 => 3 planes 1x1 16/-2 [r16/r16/r16] (g/b/r)
[   0.204][d][libmpv_render]   yuvj411p => 3 planes 4x1 8/0 [r8/r8/r8] (r/g/b)
[   0.204][d][libmpv_render]   bayer_bggr8
[   0.204][d][libmpv_render]   bayer_rggb8
[   0.204][d][libmpv_render]   bayer_gbrg8
[   0.204][d][libmpv_render]   bayer_grbg8
[   0.204][d][libmpv_render]   bayer_bggr16
[   0.204][d][libmpv_render]   bayer_bggr16be
[   0.204][d][libmpv_render]   bayer_rggb16
[   0.204][d][libmpv_render]   bayer_rggb16be
[   0.204][d][libmpv_render]   bayer_gbrg16
[   0.204][d][libmpv_render]   bayer_gbrg16be
[   0.204][d][libmpv_render]   bayer_grbg16
[   0.204][d][libmpv_render]   bayer_grbg16be
[   0.204][d][libmpv_render]   xvmc
[   0.204][d][libmpv_render]   yuv440p10 => 3 planes 1x2 16/-6 [r16/r16/r16] (r/g/b)
[   0.204][d][libmpv_render]   yuv440p10be
[   0.204][d][libmpv_render]   yuv440p12 => 3 planes 1x2 16/-4 [r16/r16/r16] (r/g/b)
[   0.204][d][libmpv_render]   yuv440p12be
[   0.204][d][libmpv_render]   ayuv64 => 1 planes 1x1 16/0 [rgba16] (argb)
[   0.204][d][libmpv_render]   ayuv64be
[   0.204][d][libmpv_render]   p010be
[   0.204][d][libmpv_render]   gbrap12be
[   0.204][d][libmpv_render]   gbrap12 => 4 planes 1x1 16/-4 [r16/r16/r16/r16] (g/b/r/a)
[   0.204][d][libmpv_render]   gbrap10be
[   0.204][d][libmpv_render]   gbrap10 => 4 planes 1x1 16/-6 [r16/r16/r16/r16] (g/b/r/a)
[   0.204][d][libmpv_render]   mediacodec
[   0.204][d][libmpv_render]   gray12be
[   0.204][d][libmpv_render]   gray12 => 1 planes 1x1 16/-4 [r16] (r)
[   0.204][d][libmpv_render]   gray10be
[   0.204][d][libmpv_render]   gray10 => 1 planes 1x1 16/-6 [r16] (r)
[   0.204][d][libmpv_render]   p016 => 2 planes 2x2 16/0 [r16/rg16] (r/gb)
[   0.204][d][libmpv_render]   p016be
[   0.204][d][libmpv_render]   d3d11
[   0.204][d][libmpv_render]   gray9be
[   0.204][d][libmpv_render]   gray9 => 1 planes 1x1 16/-7 [r16] (r)
[   0.204][d][libmpv_render]   gbrpf32be
[   0.204][d][libmpv_render]   gbrpf32 => 3 planes 1x1 32/0 [r16f/r16f/r16f] (g/b/r)
[   0.204][d][libmpv_render]   gbrapf32be
[   0.204][d][libmpv_render]   gbrapf32 => 4 planes 1x1 32/0 [r16f/r16f/r16f/r16f] (g/b/r/a)
[   0.204][d][libmpv_render]   opencl
[   0.204][d][libmpv_render]   gray14be
[   0.204][d][libmpv_render]   gray14 => 1 planes 1x1 16/-2 [r16] (r)
[   0.204][d][libmpv_render]   grayf32be
[   0.204][d][libmpv_render]   grayf32 => 1 planes 1x1 32/0 [r16f] (r)
[   0.204][d][libmpv_render]   yuva422p12be
[   0.204][d][libmpv_render]   yuva422p12 => 4 planes 2x1 16/-4 [r16/r16/r16/r16] (r/g/b/a)
[   0.204][d][libmpv_render]   yuva444p12be
[   0.204][d][libmpv_render]   yuva444p12 => 4 planes 1x1 16/-4 [r16/r16/r16/r16] (r/g/b/a)
[   0.204][d][libmpv_render]   nv42 => 2 planes 1x1 8/0 [r8/rg8] (r/bg)
[   0.204][d][libmpv_render]   vulkan
[   0.204][d][libmpv_render]   y210be
[   0.204][d][libmpv_render]   y210
[   0.204][v][libmpv_render] Testing FBO format rgba16f
[   0.204][d][libmpv_render] Resizing texture: 16x16
[   0.204][v][libmpv_render] Using FBO format rgba16f.
[   0.204][v][libmpv_render] Disabling HDR peak computation (one or more of the following is not supported: compute shaders=0, SSBO=0).
[   0.204][v][libmpv_render] No advanced processing required. Enabling dumb mode.
[   0.204][v][libmpv_render] Loading hwdec driver 'videotoolbox'
[   0.206][w][cplayer] Setting the display-fps property is deprecated; set the override-display-fps property instead.
[   0.206][d][cplayer] Setting option 'override-display-fps' = 59.940060 (flags = 0) -> 0
[   0.206][v][cplayer] Set property: display-fps=59.940060 -> 1
[   0.208][v][cplayer] Set property: icc-profile="/Library/ColorSync/Profiles/Displays/BenQ PD3200U-35B4475D-1E3D-2E94-7EB2-863948324A97.icc" -> 1
[   0.213][d][global] user path: '/Library/ColorSync/Profiles/Displays/BenQ PD3200U-35B4475D-1E3D-2E94-7EB2-863948324A97.icc' -> '/Library/ColorSync/Profiles/Displays/BenQ PD3200U-35B4475D-1E3D-2E94-7EB2-863948324A97.icc'
[   0.213][v][libmpv_render] Opening ICC profile '/Library/ColorSync/Profiles/Displays/BenQ PD3200U-35B4475D-1E3D-2E94-7EB2-863948324A97.icc'
[   0.213][d][global] user path: '/Library/ColorSync/Profiles/Displays/BenQ PD3200U-35B4475D-1E3D-2E94-7EB2-863948324A97.icc' -> '/Library/ColorSync/Profiles/Displays/BenQ PD3200U-35B4475D-1E3D-2E94-7EB2-863948324A97.icc'
[   0.213][v][bdmv/bluray] Opening /Library/ColorSync/Profiles/Displays/BenQ PD3200U-35B4475D-1E3D-2E94-7EB2-863948324A97.icc
[   0.213][v][file] Opening /Library/ColorSync/Profiles/Displays/BenQ PD3200U-35B4475D-1E3D-2E94-7EB2-863948324A97.icc
[   0.214][d][file] resize stream to 131072 bytes
[   0.214][d][file] Stream opened successfully.
[   0.214][v][libmpv_render] Testing FBO format rgba16f
[   0.214][d][libmpv_render] Resizing texture: 16x16
[   0.214][v][libmpv_render] Using FBO format rgba16f.
[   0.214][v][libmpv_render] Disabling HDR peak computation (one or more of the following is not supported: compute shaders=0, SSBO=0).
[   0.214][v][libmpv_render] No advanced processing required. Enabling dumb mode.
[   0.218][d][cplayer] Run command: loadfile, flags=64, args=["/Volumes/Storage/Movies/NVECapture.avi", "replace", ""]
[   0.218][d][global] user path: '/Users/user/Library/Application Support/com.colliderli.iina/watch_later' -> '/Users/user/Library/Application Support/com.colliderli.iina/watch_later'
[   0.218][i][cplayer] Resuming playback. This behavior can be disabled with --no-resume-playback.
[   0.218][v][cplayer] Loading config '/Users/user/Library/Application Support/com.colliderli.iina/watch_later/30DCBBB20BFAE7CA67000C3597D9F44E'
[   0.218][v][cplayer] Reading config file /Users/user/Library/Application Support/com.colliderli.iina/watch_later/30DCBBB20BFAE7CA67000C3597D9F44E
[   0.218][v][cplayer] Applying profile 'default'...
[   0.218][v][cplayer] Setting option 'start' = '9.242567' (flags = 36)
[   0.219][v][cplayer] Running hook: ytdl_hook/on_load
[   0.219][v][ytdl_hook] ytdl:// hook 
[   0.219][v][ytdl_hook] not a ytdl:// url 
[   0.219][v][bdmv/bluray] Opening /Volumes/Storage/Movies/NVECapture.avi
[   0.219][v][file] Opening /Volumes/Storage/Movies/NVECapture.avi
[   0.219][d][file] resize stream to 131072 bytes
[   0.219][d][file] Stream opened successfully.
[   0.219][v][demux] Trying demuxers for level=normal.
[   0.219][d][demux] Trying demuxer: disc (force-level: normal)
[   0.219][d][demux] Trying demuxer: edl (force-level: normal)
[   0.219][d][demux] Trying demuxer: cue (force-level: normal)
[   0.219][d][demux] Trying demuxer: rawaudio (force-level: normal)
[   0.219][d][demux] Trying demuxer: rawvideo (force-level: normal)
[   0.219][d][demux] Trying demuxer: mkv (force-level: normal)
[   0.219][d][demux] Trying demuxer: libarchive (force-level: normal)
[   0.219][d][demux] Trying demuxer: lavf (force-level: normal)
[   0.221][d][cplayer] Run command: loadfile, flags=64, args=["/Volumes/Storage/Movies/Live Serious.mp4", "append", ""]
[   0.221][d][cplayer] Run command: playlist-move, flags=64, args=["1", "0"]
[   0.224][v][lavf] Found 'avi' at score=100 size=2048.
[   0.224][v][file] stream level seek from 131072 to 484713992
[   0.224][v][file] stream level seek from 484845064 to 268
[   0.224][v][file] stream level seek from 131340 to 969395208
[   0.225][v][file] stream level seek from 969526280 to 284
[   0.225][v][file] stream level seek from 131356 to 1454219272
[   0.225][v][file] stream level seek from 1454350344 to 300
[   0.225][v][file] stream level seek from 131372 to 1938900488
[   0.225][v][file] stream level seek from 1939031560 to 316
[   0.225][v][file] stream level seek from 131388 to 2423582216
[   0.225][v][file] stream level seek from 2423713288 to 332
[   0.225][v][file] stream level seek from 131404 to 2908263432
[   0.226][v][file] stream level seek from 2908394504 to 348
[   0.226][v][file] stream level seek from 131420 to 3392945160
[   0.226][v][file] stream level seek from 3393076232 to 364
[   0.226][v][file] stream level seek from 131436 to 3877626376
[   0.226][v][file] stream level seek from 3877757448 to 380
[   0.226][v][file] stream level seek from 131452 to 4362308104
[   0.227][v][file] stream level seek from 4362439176 to 396
[   0.227][v][file] stream level seek from 131468 to 4846989320
[   0.227][v][file] stream level seek from 4847120392 to 412
[   0.227][v][file] stream level seek from 131484 to 5331670536
[   0.227][v][file] stream level seek from 5331801608 to 428
[   0.227][v][file] stream level seek from 131500 to 5816352264
[   0.227][v][file] stream level seek from 5816483336 to 444
[   0.228][v][file] stream level seek from 131516 to 6301033480
[   0.228][v][file] stream level seek from 6301164552 to 460
[   0.228][v][file] stream level seek from 131532 to 6785715208
[   0.229][v][file] stream level seek from 6785846280 to 476
[   0.229][v][file] stream level seek from 131548 to 7270396424
[   0.229][v][file] stream level seek from 7270527496 to 492
[   0.229][v][file] stream level seek from 131564 to 7755078152
[   0.230][v][file] stream level seek from 7755209224 to 508
[   0.230][v][file] stream level seek from 131580 to 8239759368
[   0.230][v][file] stream level seek from 8239890440 to 524
[   0.230][v][file] stream level seek from 131596 to 8724441096
[   0.231][v][file] stream level seek from 8724572168 to 540
[   0.231][v][file] stream level seek from 131612 to 9209122312
[   0.231][v][file] stream level seek from 9209253384 to 556
[   0.231][v][file] stream level seek from 131628 to 9693804040
[   0.231][v][file] stream level seek from 9693935112 to 572
[   0.231][v][file] stream level seek from 131644 to 10178485256
[   0.232][v][file] stream level seek from 10178616328 to 588
[   0.232][v][file] stream level seek from 131660 to 10663166472
[   0.232][v][file] stream level seek from 10663297544 to 604
[   0.232][v][file] stream level seek from 131676 to 11147848200
[   0.232][v][file] stream level seek from 11147979272 to 620
[   0.232][v][file] stream level seek from 131692 to 11632529416
[   0.232][v][file] stream level seek from 11632660488 to 636
[   0.232][v][file] stream level seek from 131708 to 12117211144
[   0.233][v][file] stream level seek from 12117342216 to 652
[   0.233][v][file] stream level seek from 131724 to 12601892360
[   0.233][v][file] stream level seek from 12602023432 to 668
[   0.233][v][file] stream level seek from 131740 to 13086574088
[   0.233][v][file] stream level seek from 13086705160 to 684
[   0.239][v][ffmpeg/demuxer] avi: Switching to NI mode, due to poor interleaving
[   0.242][v][lavf] avformat_find_stream_info() finished after 185032 bytes.
[   0.242][v][demux] Detected file format: avi (libavformat)
[   0.242][v][cplayer] Opening done: /Volumes/Storage/Movies/NVECapture.avi
[   0.242][v][cplayer] Running hook: ytdl_hook/on_preloaded
[   0.242][v][lavf] select track 0
[   0.242][v][lavf] select track 1
[   0.242][i][cplayer]  (+) Video --vid=1 (dvvideo 720x480 29.970fps)
[   0.242][i][cplayer]  (+) Audio --aid=1 (pcm_s16le 2ch 32000Hz)
[   0.242][i][cplayer]      Audio --aid=2 (pcm_s16le 2ch 32000Hz)
[   0.242][v][vo/libmpv] Assuming 59.940060 FPS for display sync.
[   0.242][v][vd] Container reported FPS: 29.970030
[   0.242][v][vd] Codec list:
[   0.242][v][vd]     dvvideo - DV (Digital Video)
[   0.242][v][vd] Opening decoder dvvideo
[   0.242][v][vd] Not trying to use hardware decoding: codec dvvideo is not on whitelist.
[   0.242][v][vd] Using software decoding.
[   0.242][v][vd] Detected 12 logical cores.
[   0.242][v][vd] Requesting 13 threads for decoding.
[   0.243][v][vd] Selected codec: dvvideo (DV (Digital Video))
[   0.243][v][vf] User filter list:
[   0.243][v][vf]   (empty)
[   0.243][v][ad] Codec list:
[   0.243][v][ad]     pcm_s16le - PCM signed 16-bit little-endian
[   0.243][v][ad] Opening decoder pcm_s16le
[   0.243][v][ad] Detected 12 logical cores.
[   0.243][v][ad] Requesting 13 threads for decoding.
[   0.243][v][ad] Selected codec: pcm_s16le (PCM signed 16-bit little-endian)
[   0.243][v][af] User filter list:
[   0.243][v][af]   (empty)
[   0.243][v][cplayer] Starting playback...
[   0.243][v][lavf] queuing seek to 9.242567
[   0.243][v][lavf] execute seek (to 9.242567 flags 32)
[   0.243][v][cplayer] hr-seek, skipping to 9.242567
[   0.243][v][file] stream level seek from 197292 to 33393664
[   0.243][v][lavf] seek done
[   0.243][v][af] [in] 32000Hz stereo 2ch s16
[   0.243][v][af] [userspeed] 32000Hz stereo 2ch s16
[   0.243][v][af] [userspeed] (disabled)
[   0.243][v][af] [convert] 32000Hz stereo 2ch s16
[   0.243][d][vd] DR parameter change to 736x480 yuv411p align=64
[   0.243][d][vd] Allocating new DR image...
[   0.243][d][vd] ...failed..
[   0.243][v][vd] DR failed - disabling.
[   0.243][v][ao] Trying audio driver 'coreaudio'
[   0.243][v][ao/coreaudio] requested format: 32000 Hz, stereo channels, s16
[   0.244][v][ao/coreaudio] selected audio output device: USB Audio CODEC  (86)
[   0.244][v][ao/coreaudio] input channel layout:
[   0.244][v][ao/coreaudio] layout: tag: <0>, bitmap: <0>, descriptions <2>
[   0.244][v][ao/coreaudio]  - description 0: label <4294967295, 64>,  flags: <0>, coords: <0.000000, 0.000000, 0.000000>
[   0.244][v][ao/coreaudio]  - description 1: label <4294967295, 64>,  flags: <0>, coords: <0.000000, 0.000000, 0.000000>
[   0.244][v][ao/coreaudio] mp chmap: unknown2
[   0.244][v][ao/coreaudio] input channel layout:
[   0.244][v][ao/coreaudio] layout: tag: <0>, bitmap: <0>, descriptions <2>
[   0.244][v][ao/coreaudio]  - description 0: label <1, 0>,  flags: <0>, coords: <0.000000, 0.000000, 0.000000>
[   0.244][v][ao/coreaudio]  - description 1: label <2, 1>,  flags: <0>, coords: <0.000000, 0.000000, 0.000000>
[   0.244][v][ao/coreaudio] mp chmap: stereo
[   0.244][v][ao/coreaudio] Channel layouts:
[   0.244][v][ao/coreaudio]  - stereo
[   0.244][v][ao/coreaudio]  - mono
[   0.244][v][ao/coreaudio] result: stereo
[   0.252][v][ao/coreaudio] Latency property cntl: 72 frames
[   0.252][v][ao/coreaudio] Latency property zisf: 512 frames
[   0.252][v][ao/coreaudio] Latency property tfas: 73 frames
[   0.252][v][ao/coreaudio] audiounit latency [us]: 500
[   0.252][v][ao/coreaudio] device latency [us]: 20531
[   0.252][v][ao/coreaudio] using soft-buffer of 6400 samples.
[   0.252][i][cplayer] AO: [coreaudio] 32000Hz stereo 2ch s16
[   0.252][v][cplayer] AO: Description: CoreAudio AudioUnit
[   0.253][v][af] [convert] (disabled)
[   0.253][v][af] [out] 32000Hz stereo 2ch s16
[   0.253][v][vd] Decoder format: 720x480 [8:9] yuv411p auto/auto/auto/auto/auto CL=unknown
[   0.253][v][vd] Using container aspect ratio.
[   0.253][v][vf] [in] 720x480 [8:9] yuv411p bt.601/bt.601-525/bt.1886/limited/display SP=1.000000 CL=mpeg2/4/h264
[   0.253][v][vf] [userdeint] 720x480 [8:9] yuv411p bt.601/bt.601-525/bt.1886/limited/display SP=1.000000 CL=mpeg2/4/h264
[   0.253][v][vf] [userdeint] (disabled)
[   0.253][v][vf] [autorotate] 720x480 [8:9] yuv411p bt.601/bt.601-525/bt.1886/limited/display SP=1.000000 CL=mpeg2/4/h264
[   0.253][v][vf] [autorotate] (disabled)
[   0.253][v][vf] [convert] 720x480 [8:9] yuv411p bt.601/bt.601-525/bt.1886/limited/display SP=1.000000 CL=mpeg2/4/h264
[   0.253][v][vf] [convert] (disabled)
[   0.253][v][vf] [out] 720x480 [8:9] yuv411p bt.601/bt.601-525/bt.1886/limited/display SP=1.000000 CL=mpeg2/4/h264
[   0.253][i][cplayer] VO: [libmpv] 720x480 => 720x540 yuv411p
[   0.253][v][cplayer] VO: Description: render API for libmpv
[   0.253][v][vo/libmpv] reconfig to 720x480 [8:9] yuv411p bt.601/bt.601-525/bt.1886/limited/display SP=1.000000 CL=mpeg2/4/h264
[   0.254][v][libmpv_render] Window size: 1280x720 (Borders: l=0 t=0 r=0 b=0)
[   0.254][v][libmpv_render] Video source: 720x480 (8:9)
[   0.254][v][libmpv_render] Video display: (0, 0) 720x480 -> (0, 0) 1280x720
[   0.254][v][libmpv_render] Video scale: 1.777778/1.500000
[   0.254][v][libmpv_render] OSD borders: l=0 t=0 r=0 b=0
[   0.254][v][libmpv_render] Video borders: l=0 t=0 r=0 b=0
[   0.254][v][libmpv_render] Texture for plane 0: 720x480
[   0.254][v][libmpv_render] Texture for plane 1: 180x480
[   0.254][v][libmpv_render] Texture for plane 2: 180x480
[   0.254][v][libmpv_render] Testing FBO format rgba16f
[   0.254][d][libmpv_render] Resizing texture: 16x16
[   0.254][v][libmpv_render] Using FBO format rgba16f.
[   0.254][v][libmpv_render] No advanced processing required. Enabling dumb mode.
[   0.256][v][libmpv_render] Detected ICC profile contrast: inf
[   0.256][w][libmpv_render] ICC profile detected contrast very high (>100000), falling back to contrast 1000 for sanity. Set the icc-contrast option to silence this warning.
[   0.267][v][cplayer] first video frame after restart shown
[   0.267][v][cplayer] audio sync: sync_to_video=1, offset=-0.000000
[   0.267][v][cplayer] starting audio playback
[   0.295][d][libmpv_render] vertex shader source:
[   0.295][d][libmpv_render] [  1] #version 410
[   0.295][d][libmpv_render] [  2] #define tex1D texture
[   0.295][d][libmpv_render] [  3] #define tex3D texture
[   0.295][d][libmpv_render] [  4] #define LUT_POS(x, lut_size) mix(0.5 / (lut_size), 1.0 - 0.5 / (lut_size), (x))
[   0.295][d][libmpv_render] [  5] in vec2 vertex_position;
[   0.295][d][libmpv_render] [  6] in vec2 vertex_texcoord0;
[   0.295][d][libmpv_render] [  7] out vec2 texcoord0;
[   0.295][d][libmpv_render] [  8] in vec2 vertex_texcoord1;
[   0.295][d][libmpv_render] [  9] out vec2 texcoord1;
[   0.295][d][libmpv_render] [ 10] in vec2 vertex_texcoord2;
[   0.295][d][libmpv_render] [ 11] out vec2 texcoord2;
[   0.295][d][libmpv_render] [ 12] void main() {
[   0.295][d][libmpv_render] [ 13] gl_Position = vec4(vertex_position, 1.0, 1.0);
[   0.295][d][libmpv_render] [ 14] texcoord0 = vertex_texcoord0;
[   0.295][d][libmpv_render] [ 15] texcoord1 = vertex_texcoord1;
[   0.295][d][libmpv_render] [ 16] texcoord2 = vertex_texcoord2;
[   0.295][d][libmpv_render] [ 17] }
[   0.296][d][libmpv_render] fragment shader source:
[   0.296][d][libmpv_render] [  1] #version 410
[   0.296][d][libmpv_render] [  2] #define tex1D texture
[   0.296][d][libmpv_render] [  3] #define tex3D texture
[   0.296][d][libmpv_render] [  4] #define LUT_POS(x, lut_size) mix(0.5 / (lut_size), 1.0 - 0.5 / (lut_size), (x))
[   0.296][d][libmpv_render] [  5] out vec4 out_color;
[   0.296][d][libmpv_render] [  6] in vec2 texcoord0;
[   0.296][d][libmpv_render] [  7] in vec2 texcoord1;
[   0.296][d][libmpv_render] [  8] in vec2 texcoord2;
[   0.296][d][libmpv_render] [  9] uniform mat3 colormatrix;
[   0.296][d][libmpv_render] [ 10] uniform vec3 colormatrix_c;
[   0.296][d][libmpv_render] [ 11] uniform vec3 src_luma;
[   0.296][d][libmpv_render] [ 12] uniform vec3 dst_luma;
[   0.296][d][libmpv_render] [ 13] uniform sampler3D lut_3d;
[   0.296][d][libmpv_render] [ 14] uniform sampler2D texture0;
[   0.296][d][libmpv_render] [ 15] uniform vec2 texture_size0;
[   0.296][d][libmpv_render] [ 16] uniform mat2 texture_rot0;
[   0.296][d][libmpv_render] [ 17] uniform vec2 texture_off0;
[   0.296][d][libmpv_render] [ 18] uniform vec2 pixel_size0;
[   0.296][d][libmpv_render] [ 19] uniform sampler2D texture1;
[   0.296][d][libmpv_render] [ 20] uniform vec2 texture_size1;
[   0.296][d][libmpv_render] [ 21] uniform mat2 texture_rot1;
[   0.297][d][libmpv_render] [ 22] uniform vec2 texture_off1;
[   0.297][d][libmpv_render] [ 23] uniform vec2 pixel_size1;
[   0.297][d][libmpv_render] [ 24] uniform sampler2D texture2;
[   0.297][d][libmpv_render] [ 25] uniform vec2 texture_size2;
[   0.297][d][libmpv_render] [ 26] uniform mat2 texture_rot2;
[   0.297][d][libmpv_render] [ 27] uniform vec2 texture_off2;
[   0.297][d][libmpv_render] [ 28] uniform vec2 pixel_size2;
[   0.297][d][libmpv_render] [ 29] void main() {
[   0.297][d][libmpv_render] [ 30] vec4 color = vec4(0.0, 0.0, 0.0, 1.0);
[   0.297][d][libmpv_render] [ 31] color.r = 1.000000 * vec4(texture(texture0, texcoord0)).r;
[   0.297][d][libmpv_render] [ 32] color.g = 1.000000 * vec4(texture(texture1, texcoord1)).r;
[   0.297][d][libmpv_render] [ 33] color.b = 1.000000 * vec4(texture(texture2, texcoord2)).r;
[   0.297][d][libmpv_render] [ 34] color = color.rgbr;
[   0.297][d][libmpv_render] [ 35] color.rgb = mat3(colormatrix) * color.rgb + colormatrix_c;
[   0.297][d][libmpv_render] [ 36] color.a = 1.0;
[   0.297][d][libmpv_render] [ 37] // color mapping
[   0.297][d][libmpv_render] [ 38] color.rgb *= vec3(1.000000);
[   0.297][d][libmpv_render] [ 39] color.rgb *= vec3(1.000000);
[   0.297][d][libmpv_render] [ 40] vec3 cpos;
[   0.297][d][libmpv_render] [ 41] cpos[0] = LUT_POS(color[0], 64.0);
[   0.297][d][libmpv_render] [ 42] cpos[1] = LUT_POS(color[1], 64.0);
[   0.297][d][libmpv_render] [ 43] cpos[2] = LUT_POS(color[2], 64.0);
[   0.297][d][libmpv_render] [ 44] color.rgb = tex3D(lut_3d, cpos).rgb;
[   0.297][d][libmpv_render] [ 45] out_color = color;
[   0.297][d][libmpv_render] [ 46] }
[   0.297][d][libmpv_render] shader link log (status=1): 
[   0.675][v][cplayer] playback restart complete @ 9.242567
[   0.701][v][cplayer] Set property: pause=true -> 1
[   0.705][v][cplayer] Set property: pause=false -> 1
[   0.768][v][libmpv_render] Window size: 1244x708 (Borders: l=0 t=0 r=0 b=0)
[   0.768][v][libmpv_render] Video source: 720x480 (8:9)
[   0.768][v][libmpv_render] Video display: (0, 0) 720x480 -> (0, 0) 1244x708
[   0.768][v][libmpv_render] Video scale: 1.727778/1.475000
[   0.768][v][libmpv_render] OSD borders: l=0 t=0 r=0 b=0
[   0.768][v][libmpv_render] Video borders: l=0 t=0 r=0 b=0
[   0.821][v][libmpv_render] Window size: 1152x680 (Borders: l=0 t=0 r=0 b=0)
[   0.821][v][libmpv_render] Video source: 720x480 (8:9)
[   0.821][v][libmpv_render] Video display: (0, 0) 720x480 -> (0, 0) 1152x680
[   0.821][v][libmpv_render] Video scale: 1.600000/1.416667
[   0.821][v][libmpv_render] OSD borders: l=0 t=0 r=0 b=0
[   0.821][v][libmpv_render] Video borders: l=0 t=0 r=0 b=0
[   0.856][v][libmpv_render] Window size: 1068x652 (Borders: l=0 t=0 r=0 b=0)
[   0.856][v][libmpv_render] Video source: 720x480 (8:9)
[   0.856][v][libmpv_render] Video display: (0, 0) 720x480 -> (0, 0) 1068x652
[   0.856][v][libmpv_render] Video scale: 1.483333/1.358333
[   0.856][v][libmpv_render] OSD borders: l=0 t=0 r=0 b=0
[   0.856][v][libmpv_render] Video borders: l=0 t=0 r=0 b=0
[   0.886][v][libmpv_render] Window size: 976x624 (Borders: l=0 t=0 r=0 b=0)
[   0.886][v][libmpv_render] Video source: 720x480 (8:9)
[   0.886][v][libmpv_render] Video display: (0, 0) 720x480 -> (0, 0) 976x624
[   0.886][v][libmpv_render] Video scale: 1.355556/1.300000
[   0.886][v][libmpv_render] OSD borders: l=0 t=0 r=0 b=0
[   0.886][v][libmpv_render] Video borders: l=0 t=0 r=0 b=0
[   0.922][v][libmpv_render] Window size: 892x596 (Borders: l=0 t=0 r=0 b=0)
[   0.922][v][libmpv_render] Video source: 720x480 (8:9)
[   0.922][v][libmpv_render] Video display: (0, 0) 720x480 -> (0, 0) 892x596
[   0.922][v][libmpv_render] Video scale: 1.238889/1.241667
[   0.922][v][libmpv_render] OSD borders: l=0 t=0 r=0 b=0
[   0.922][v][libmpv_render] Video borders: l=0 t=0 r=0 b=0
[   0.953][v][libmpv_render] Window size: 816x572 (Borders: l=0 t=0 r=0 b=0)
[   0.953][v][libmpv_render] Video source: 720x480 (8:9)
[   0.953][v][libmpv_render] Video display: (0, 0) 720x480 -> (0, 0) 816x572
[   0.953][v][libmpv_render] Video scale: 1.133333/1.191667
[   0.953][v][libmpv_render] OSD borders: l=0 t=0 r=0 b=0
[   0.953][v][libmpv_render] Video borders: l=0 t=0 r=0 b=0
[   0.987][v][libmpv_render] Window size: 760x552 (Borders: l=0 t=0 r=0 b=0)
[   0.987][v][libmpv_render] Video source: 720x480 (8:9)
[   0.988][v][libmpv_render] Video display: (0, 0) 720x480 -> (0, 0) 760x552
[   0.988][v][libmpv_render] Video scale: 1.055556/1.150000
[   0.988][v][libmpv_render] OSD borders: l=0 t=0 r=0 b=0
[   0.988][v][libmpv_render] Video borders: l=0 t=0 r=0 b=0
[   1.186][v][libmpv_render] Window size: 740x548 (Borders: l=0 t=0 r=0 b=0)
[   1.186][v][libmpv_render] Video source: 720x480 (8:9)
[   1.186][v][libmpv_render] Video display: (0, 0) 720x480 -> (0, 0) 740x548
[   1.186][v][libmpv_render] Video scale: 1.027778/1.141667
[   1.186][v][libmpv_render] OSD borders: l=0 t=0 r=0 b=0
[   1.186][v][libmpv_render] Video borders: l=0 t=0 r=0 b=0

```
</details>

<br>

**Steps to reproduce:**
Open a video file. So far, I've had the problem on both H264 MPEG4 files of various resolutions and orientations, and an ancient .avi export from a video camera.

- [x] MPV does not have this problem.

**How often does this happen?**
Inconsistent, but more consistently with some files than others. Some always hang. Some sometimes. Many never.

**Comment by ksec** [comment]
This happen mostly to video in portrait mode. And it only happen after Big Sur update.

**Comment by abatty** [comment]
Yes, now that you mention it, I do seem to have more problems with portrait videos. And yes, they work find in mpv. It's very strange.

**Comment by ksec** [comment]
It has to do with Resizing Windows on Big Sur. 

In Preference > UI > Resize Windows to Fit Video Size >

If you disable it then it will work fine. Wondering if dev are taking any notice.

**Comment by abatty** [comment]
That preference change fixed it! I hope the devs can address it soon. I was totally lost.

**Comment by jisuoqing** [comment]
> It has to do with Resizing Windows on Big Sur.
> 
> In Preference > UI > Resize Windows to Fit Video Size >
> 
> If you disable it then it will work fine. Just went on TG channel Dev is current busy with other work. So we might have to wait for this to be fixed.

Thanks for this solution. But even if this option is disabled, when you drag the window edge to resize it for a few times, IINA still hangs forever / crashes. So maybe we have to wait for the bug being fixed.

**Comment by abatty** [comment]
>But even if this option is disabled, when you drag the window edge to resize it for a few times, IINA still hangs forever / crashes.

I can confirm. It hangs as soon as I change the size of the window. Bummer.

**Comment by ksec** [comment]
> > It has to do with Resizing Windows on Big Sur.
> > In Preference > UI > Resize Windows to Fit Video Size >
> > If you disable it then it will work fine. Just went on TG channel Dev is current busy with other work. So we might have to wait for this to be fixed.
> 
> Thanks for this solution. But even if this option is disabled, when you drag the window edge to resize it for a few times, IINA still hangs forever / crashes. So maybe we have to wait for the bug being fixed.

Confirm as well, but changing video size ( 1x , 2x, Full Screen ) works fine. 

Again, tested on mpv 0.32, 0.33

**Comment by lucasff** [comment]
I never noticed that it is because of resizing the windows, but I do notice when I'm playing multiple videos, and suddenly opening one hangs IINA completely. Bummer!

**Comment by treasure07** [comment]
到底还解不解决了

**Comment by jisuoqing** [comment]
Btw, I downgraded to 1.0.6 and it works fine.

**Comment by zhaoquntao** [comment]
我已经把 电脑(MacPro)系统升级到最新了  还是会出现 快进/退 卡顿的问题

**Comment by jisuoqing** [comment]
This problem went away with macos 12.0.1.

**Comment by zhaoquntao** [comment]
> This problem went away with macos 12.0.1.
然后并没有 我的inter版 MacPro  系统是 12.0.1    刚刚完全卸载IINA  重新安装 还是出现 快进/退  转圈圈问题

**Comment by low-batt** [comment]
The fix has been merged into the develop branch in commit https://github.com/iina/iina/commit/39be32d8c46b7b4b681473f58f6f0dd8f967c324

Apple has fixed this macOS Big Sur issue in Monterey.
